src/main/scala/cromwell/engine/ExecutionStatusStore.scala

```diff
@@ -0,0 +1,12 @@
+package cromwell.engine
+
+import cromwell.binding.{Call, WdlBinding}
+
+/**
+ * Corresponds to the "execution table" of our discussions.
+ */
+class ExecutionStatusStore(binding: WdlBinding) {
+  /** FIXME this implementation only sorta works for "hello world" and is
+    * FIXME not adequate for Sprint 2/3. */
+  def runnableCalls: Seq[Call] = binding.workflows.head.calls
+}
```

Comment:
Per HipChat discussion this is an old pre-rename version of what is now `ExecutionStore`, and hopefully can just be deleted.